src/main/kotlin/game/racing/domain/RacingGame.kt

```diff
@@ -0,0 +1,21 @@
+package game.racing.domain
+
+import game.racing.view.InputView
+import game.racing.view.ResultView
+
+class RacingGame(private val carManger: CarManger) {
+    fun start() {
+        val tryNumber = InputView.inputTryNumber()
+
+        ResultView.printResultIntroduction()
+        repeat(tryNumber) {
+            carManger.moveCars()
+            ResultView.printResult(carManger.cars)
+        }
+    }
+}
+
+fun main() {
+    val racingGame = RacingGame(CarManger(CarFactory.generateCarByNumberOfCar(InputView.inputNumberOfCar())))
```

Comment:
적절하게 변수로 추출하면 조금 읽기쉬운 코드가 될 것 같아요. 
```suggestion
    val cars = CarFactory.generateCarByNumberOfCar(InputView.inputNumberOfCar())
    val racingGame = RacingGame(CarManger(cars))
```